docarray/typing/tensor/video/video_tensor_mixin.py

```diff
@@ -0,0 +1,118 @@
+from typing import TYPE_CHECKING, BinaryIO, Optional, Type, TypeVar, Union
+
+import numpy as np
+
+from docarray.typing.tensor.audio.audio_tensor import AudioTensor
+
+if TYPE_CHECKING:
+    from docarray.typing import VideoNdArray, VideoTorchTensor
+
+
+T = TypeVar('T', bound='VideoTensorMixin')
+
+
+class VideoTensorMixin:
+    @staticmethod
```

Comment:
If you wan to use a specific superclass implementation, instead of using `super()` you can do something like:

```python
SuperclassThatYouWant.foo(*inputs)
```
Or if you need to pass the subclass `cls`:
```python
SuperclassThatYouWant.foo.__func__(cls, *inputs)
```
This unbinds `foo` from the class, so you can give it any `cls` argument that you want.

You can see this in the `__class_getitem__` implementation of `NdArray`.